commcare_connect/opportunity/management/commands/setup_logical_replication.py

```diff
@@ -0,0 +1,120 @@
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+from django.db import DEFAULT_DB_ALIAS, connections
+
+from commcare_connect.opportunity.models import (
+    Assessment,
+    CompletedModule,
+    CompletedWork,
+    LearnModule,
+    Opportunity,
+    OpportunityAccess,
+    Payment,
+    PaymentUnit,
+    UserVisit,
+)
+from commcare_connect.users.models import User
+
+REPLICATION_ALLOWED_MODELS = [
+    Opportunity,
+    OpportunityAccess,
+    LearnModule,
+    CompletedModule,
+    Payment,
+    User,
+    UserVisit,
+    CompletedWork,
+    PaymentUnit,
+    Assessment,
+]
+
+
+class Command(BaseCommand):
```

Comment:
I have set this up as a management command instead of a migration as that allows us to run this only on envs where it's needed.